### PR TITLE
Ss/local vols fix

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

**Comment by joel-hamill** [comment]
@sascala made some edits...LGTM. go ahead an merge after you review. Beware of the passive voice.

File: 1.7/administration/storage/persistent-volume.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+## Benefits of using local persistent volumes
+
+- All resources needed to run tasks of your stateful service are dynamically reserved, thus ensuring the ability to relaunch the task on the same node using the same volume when needed.
+- You don't need constraints to pin a task to a particular agent where its data resides
+- You can still use constraints to specify distribution logic
+- Marathon lets you locate and destroy an unused persistent volume if you don't need it anymore
+
+# Create an application with local persistent volumes
+
+## Prerequisites
```
Comment (pyronicide) [inline 1.7/administration/storage/persistent-volume.md]: These are for vanilla mesos. We have different ones for DC/OS.

File: 1.7/administration/storage/persistent-volume.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+In order to create stateful applications using local persistent volumes in Marathon, you need to set 2 command line flags that Marathon will use to reserve/unreserve resources and create/destroy volumes.
+
+- `--mesos_authentication_principal`: You can choose any that suits your needs. However, if you have set up ACLs on your Mesos master, this must be an authenticated and authorized prinicpal.
+- `--mesos_role`: This should be a unique role and will be used implicitly, that is, you don't need to configure the Mesos master via `--roles`.
+
+## Configuration options
+
+Configure a persistent volume with the following options:
+
+```json
```
Comment (pyronicide) [inline 1.7/administration/storage/persistent-volume.md]: How about a simple example?

File: 1.7/administration/storage/persistent-volume.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+
+If reserving resources or creating persistent volumes fails, the created task will timeout after the configured `task_reservation_timeout` (default: 20 seconds) and a new reservation attempt will be made. In case a task is `LOST` (because its agent is disconnected or crashed), the reservations and volumes will not timeout and you need to manually delete and wipe the task in order to let Marathon launch a new one.
+
+# Potential Pitfalls
+
+Be aware of the following issues and limitations when using stateful applications in Marathon that make use of dynamic resevations and persistent volumes.
+
+## Static Reservations
+
+Dynamic reservations can only be created for unreserved resources. If you specify an agent's resources to be reserved for a role via the Mesos `--resources` or `--default_role` flag, these resources cannot be used for dynamic reservations. In addition, if Marathon is started with the `--default_accepted_resource_roles` flag specifying a value that does not contain `*`, your application definition should explicitly specify `"acceptedResourceRoles": ["*"]` in order to allow usage and reservation of unreserved cluster resources.
```
Comment (pyronicide) [inline 1.7/administration/storage/persistent-volume.md]: This is for vanilla mesos only.

File: 1.7/administration/storage/persistent-volume.md (hunk 1 above)
Comment (sascala) [inline 1.7/administration/storage/persistent-volume.md]: What would the DC/OS prereqs be?

Comment (joel-hamill) [inline 1.7/administration/storage/persistent-volume.md]: @sascala these https://dcos.io/docs/latest/administration/installing/custom/system-requirements/ ?

Comment (pyronicide) [inline 1.7/administration/storage/persistent-volume.md]: Need to make sure the hosts are setup correctly, I'm not sure outside of that.

**Pull request merged**